helm/defectdojo/templates/configmap.yaml

```diff
@@ -16,7 +16,7 @@ data:
   DD_ALLOWED_HOSTS: {{ .Values.host }}
   DD_CELERY_BROKER_SCHEME: {{ if eq .Values.celery.broker "rabbitmq" }}amqp{{ end }}{{ if eq .Values.celery.broker "redis" }}redis{{ end }}
   DD_CELERY_BROKER_USER: '{{ if eq .Values.celery.broker "rabbitmq" }}user{{ end }}'
-  DD_CELERY_BROKER_HOST: {{ $fullName }}-{{ if eq .Values.celery.broker "rabbitmq" }}rabbitmq{{ end }}{{ if eq .Values.celery.broker "redis" }}redis-master{{ end }}
+  DD_CELERY_BROKER_HOST: {{ .Release.Name }}-{{ if eq .Values.celery.broker "rabbitmq" }}rabbitmq{{ end }}{{ if eq .Values.celery.broker "redis" }}redis-master{{ end }}
```

Comment:
Was it your intent to change the name here? If yes, can you please shortly explain why?

Reply:
The $fullname does appear to be the correct name for the broker host but rather the `.Releese.Name`.